src/value/borrowed/cmp.rs

```diff
@@ -1,101 +1,127 @@
 use super::Value;
+use crate::{OwnedValue, ValueTrait};
+use float_cmp::approx_eq;
 
-impl<'a> PartialEq<()> for Value<'a> {
-    fn eq(&self, _other: &()) -> bool {
-        if let Value::Null = self {
-            true
-        } else {
-            false
+impl<'a> PartialEq for Value<'a> {
+    fn eq(&self, other: &Self) -> bool {
```

Comment:
does this automatically get `#[inline]`?

Reply:
Ja rustc is pretty smart about inlining functions. We only use it in other places because we think we know better 😂